### ISSUE TITLE
force_delete under user resource is not working

### ISSUE THREAD

**Issue by Ajaiswal2610** [awlsring/terraform-provider-headscale#9]
Issue: 
I am getting error "user has devices attached to it. Set `force_delete` to `true` to delete the user anyway " even when i have already set force_delete to true . 

```
resource "headscale_user" "chainlake" {
    for_each = var.hcloud_vms
    name    = each.value.id
    force_delete = true
}
```

am i missing something ?? seems like the attribute force_delete is not ready in code . 

if i revert the code for user resource terraform fails throwing the error:
```
Provider produced inconsistent result after apply
When applying changes to headscale_user.chainlake["2"], provider
provider[\"registry.terraform.io/awlsring/headscale\"]" produced an
unexpected new value: .force_delete: was cty.True, but now null.
This is a bug in the provider, which should be reported in the provider's
```

**Comment by awlsring** [comment]
Sorry, I missed this usecase. I had only tested updating a resource and not setting this field by default.

I got a couple changes, I'll push a fix.

**Comment by awlsring** [comment]
Ive published version 0.1.9, try it out and let me know if it fixes your problems.

**Comment by Ajaiswal2610** [comment]
i still can't force delete the users those were created with earlier plugin version , it still says m[1m[31mError: [0m[0m[1mError deleting user ,User has devices attached to it. Set `force_delete` to `true` to delete the user anyway. even after setting force_delete = true. 



`resource "headscale_user" "chainlake" {
    for_each = var.hcloud_vms
    name    = each.value.id
    force_delete = true
}`
